src/redux/greetings/greetingsSlice.js

```diff
@@ -1,28 +1,28 @@
-import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
 
-const ApiData = "http://localhost:3000/random_greeting";
```

Comment:
- [ ]  I have run your project and I found that the same issue previous reviewer mentioned is still there. I checked in the console and I found that User can not access the data from the api due to CORS.
![image](https://user-images.githubusercontent.com/43877689/211150091-0037fb69-cc60-4a22-9ee8-679b6ead6bcf.png)

Please check on this resources so that you can fix the issue. 
- https://www.youtube.com/watch?v=C_O_bvE_j6M
- https://www.youtube.com/watch?v=Sv8442cL_q4&t=143s

This will make your project work run properly.